dev-python/pyzbar/pyzbar-0.1.8.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Read one-dimensional barcodes and QR codes from Python"
+HOMEPAGE="https://github.com/NaturalHistoryMuseum/pyzbar/"
+SRC_URI="https://github.com/NaturalHistoryMuseum/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/pillow[${PYTHON_USEDEP}]
+	media-gfx/zbar
+	virtual/python-enum34[${PYTHON_USEDEP}]
+	virtual/python-pathlib[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		virtual/python-unittest-mock[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+"
+
+python_test() {
+	"${EPYTHON}" -m unittest discover -v || die "Testing failed with ${EPYTHON}"
+}
```

Comment:
Can this be replaced with `distutils_enable_tests`?

Reply:
okay

Reply:
```bash
>>> Source compiled.
Unable to configure loopback interface: Operation not permitted
>>> Test phase: dev-python/pyzbar-0.1.8
 * python3_6: running distutils-r1_run_phase python_test
test_bounding_box (pyzbar.tests.test_locations.TestLocations) ... ok
test_convex_duplicates (pyzbar.tests.test_locations.TestLocations) ... ok
test_convex_hull_empty (pyzbar.tests.test_locations.TestLocations) ... ok
test_convex_square (pyzbar.tests.test_locations.TestLocations) ... ok
test_other (pyzbar.tests.test_locations.TestLocations) ... ok
test_decode_code128 (pyzbar.tests.test_pyzbar.TestDecode)
Read both barcodes in `code128.png` ... ok
test_decode_numpy (pyzbar.tests.test_pyzbar.TestDecode)
Read image using Pillow and convert to numpy.ndarray ... ok
test_decode_opencv (pyzbar.tests.test_pyzbar.TestDecode)
Read image using OpenCV ... skipped 'OpenCV not installed'
test_decode_qrcode (pyzbar.tests.test_pyzbar.TestDecode)
Read barcode in `qrcode.png` ... ok
test_decode_qrcode_rotated (pyzbar.tests.test_pyzbar.TestDecode)
Read barcode in `qrcode_rotated.png` ... ok
test_decode_tuple (pyzbar.tests.test_pyzbar.TestDecode)
Read barcodes in pixels ... ok
test_empty (pyzbar.tests.test_pyzbar.TestDecode)
Do not show any output for an image that does not contain a barcode ... ok
test_external_dependencies (pyzbar.tests.test_pyzbar.TestDecode)
External dependencies ... ok
test_inconsistent_dimensions (pyzbar.tests.test_pyzbar.TestDecode) ... /var/tmp/portage/dev-python/pyzbar-0.1.8/work/pyzbar-0.1.8/pyzbar/tests/test_pyzbar.py:188: DeprecationWarning: Please use assertRaisesRegex instead.
  decode, data
ok
test_symbols (pyzbar.tests.test_pyzbar.TestDecode)
Read only qrcodes in `qrcode.png` ... ok
test_symbols_not_present (pyzbar.tests.test_pyzbar.TestDecode)
Read only code128 in `qrcode.png` ... ok
test_unsupported_bits_per_pixel (pyzbar.tests.test_pyzbar.TestDecode) ... ok
test_unsupported_bpp (pyzbar.tests.test_pyzbar.TestDecode) ... ok
test_zbar_image_create_fail (pyzbar.tests.test_pyzbar.TestDecode) ... ok
test_zbar_image_scanner_create_fail (pyzbar.tests.test_pyzbar.TestDecode) ... ok
test_zbar_scan_image_fail (pyzbar.tests.test_pyzbar.TestDecode) ... ok
test_read_code128 (pyzbar.tests.test_read_zbar.TestReadZbar)
Read CODE 128 barcodes ... ok
test_read_qrcode (pyzbar.tests.test_read_zbar.TestReadZbar)
Read QR code ... ok
test_found_non_windows (pyzbar.tests.test_zbar_library.TestLoad)
zbar loaded ok on non-Windows platform ... ok
test_found_second_attempt_windows (pyzbar.tests.test_zbar_library.TestLoad)
zbar found on the second attempt on Windows ... ok
test_found_windows (pyzbar.tests.test_zbar_library.TestLoad)
zbar found on Windows ... ok
test_not_found_non_windows (pyzbar.tests.test_zbar_library.TestLoad)
zbar not found on non-Windows platform ... ok
test_not_found_windows (pyzbar.tests.test_zbar_library.TestLoad)
zbar not found on Windows ... ok
test_32bit (pyzbar.tests.test_zbar_library.TestWindowsFnames) ... ok
test_64bit (pyzbar.tests.test_zbar_library.TestWindowsFnames) ... ok

----------------------------------------------------------------------
Ran 30 tests in 0.334s

OK (skipped=1)
 * python3_6: running distutils-r1_run_phase _distutils-r1_clean_egg_info
>>> Completed testing dev-python/pyzbar-0.1.8
```